### DISCUSSION TITLE
zh/guide/key-use

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#766] [Announcements]
# zh/guide/key-use

验证使用教程 找到★⋅micah-bot-verify⋅★频道 直链 (https://discord.com/channels/1069057220802781265/1203687333107335198); 找到Micah-bot所发的这则消息; img.png 根据提示的流程来; 如果你是2024年2月4日之前的Sponsor,点击赞助者补偿,...

https://korepi.com/zh/guide/key-use.html

**Comment by SYCXyyds666** [comment]
So where does card get it?
